NvAPIWrapper/Native/Display/Structures/HDRColorDataV2.cs

```diff
@@ -0,0 +1,140 @@
+        internal BPC _BPC;
+
+        public HDRColorDataV2(
+            HDRCommand hdrCommand,
+            HDRMode hdrMode,
+            StaticMetadataDescriptorId staticMetadataDescriptorId,
+            MasteringDisplayData masteringDisplayData,
+            ColorFormat colorFormat,
+            HDRDynamicRange hdrDynamicRange,
+            BPC bpc)
+        {
+            this = typeof(HDRColorDataV2).Instantiate<HDRColorDataV2>();
+            _HDRCommand = hdrCommand;
+            _HDRMode = hdrMode;
+            _StaticMetadataDescriptorId = staticMetadataDescriptorId;
+            _MasteringDisplayData = masteringDisplayData;
+            _ColorFormat = colorFormat;
+            _HDRDynamicRange = hdrDynamicRange;
+            _BPC = bpc;
+        }
+
+        public HDRColorDataV2(
+            HDRCommand hdrCommand,
+            HDRMode hdrMode,
+            StaticMetadataDescriptorId staticMetadataDescriptorId,
+            MasteringDisplayData masteringDisplayData,
+            ColorFormat colorFormat,
+            HDRDynamicRange hdrDynamicRange)
+            : this(hdrCommand, hdrMode, staticMetadataDescriptorId, masteringDisplayData, colorFormat, hdrDynamicRange, BPC.Default)
+        {
+        }
+
+        public HDRColorDataV2(
+            HDRCommand hdrCommand,
+            HDRMode hdrMode,
+            StaticMetadataDescriptorId staticMetadataDescriptorId,
+            MasteringDisplayData masteringDisplayData,
+            ColorFormat colorFormat)
+            : this(hdrCommand, hdrMode, staticMetadataDescriptorId, masteringDisplayData, colorFormat, HDRDynamicRange.Auto, BPC.Default)
+        {
+        }
+
+        public HDRColorDataV2(
+            HDRCommand hdrCommand,
+            HDRMode hdrMode,
+            StaticMetadataDescriptorId staticMetadataDescriptorId,
+            MasteringDisplayData masteringDisplayData)
+            : this(hdrCommand, hdrMode, staticMetadataDescriptorId, masteringDisplayData, ColorFormat.Default, HDRDynamicRange.Auto, BPC.Default)
+        {
+        }
+
+        public HDRColorDataV2(HDRCommand hdrCommand, HDRMode hdrMode, StaticMetadataDescriptorId staticMetadataDescriptorId)
+            : this(hdrCommand, hdrMode, staticMetadataDescriptorId, default, ColorFormat.Default, HDRDynamicRange.Auto, BPC.Default)
+        {
+        }
+
+        public HDRColorDataV2(HDRCommand hdrCommand, HDRMode hdrMode)
+            : this(hdrCommand, hdrMode, StaticMetadataDescriptorId.StaticMetadataType1, default, ColorFormat.Default, HDRDynamicRange.Auto, BPC.Default)
+        {
+        }
+
+        public HDRColorDataV2(HDRCommand hdrCommand)
+            : this(hdrCommand, HDRMode.Off, StaticMetadataDescriptorId.StaticMetadataType1, default, ColorFormat.Default, HDRDynamicRange.Auto, BPC.Default)
+        {
+        }
+
+        /// <inheritdoc />
+        public HDRCommand HDRCommand
+        {
+            get => _HDRCommand;
+            set => _HDRCommand = value;
```

Comment:
No setter for these or similar properties as well. Since these values should be provided by the constructor. I think it is better to keep all structures immutable in the library unless for internal use.